var/spack/repos/builtin/packages/brahma/package.py

```diff
@@ -15,20 +15,31 @@ class Brahma(CMakePackage):
 
     license("MIT")
 
-    version("develop", branch="dev")
+    version("develop", branch="develop")
     version("master", branch="master")
+    version("0.0.5", tag="v0.0.5", commit="219198c653cc4add845a644872e7b963a8de0fe2")
+    version("0.0.4", tag="v0.0.4", commit="8f41cc885dd8e31a1f134cbbcbaaab7e5d84331e")
     version("0.0.3", tag="v0.0.3", commit="fd201c653e8fa00d4ba6197a56a513f740e3014e")
     version("0.0.2", tag="v0.0.2", commit="4a36d5c08787d41c939fa1b987344b69d9ef97a6")
     version("0.0.1", tag="v0.0.1", commit="15156036f14e36511dfc3f3751dc953540526a2b")
 
     variant("mpi", default=False, description="Enable MPI support")
     depends_on("cpp-logger@0.0.1", when="@:0.0.1")
-    depends_on("cpp-logger@0.0.2:", when="@0.0.2:")
+    depends_on("cpp-logger@0.0.2", when="@0.0.2:")
+    depends_on("cpp-logger@0.0.3", when="@0.0.4")
+    depends_on("cpp-logger@0.0.4", when="@0.0.5")
+    depends_on("cpp-logger@0.0.4", when="@develop")
     depends_on("gotcha@1.0.4", when="@:0.0.1")
-    depends_on("gotcha@1.0.5:", when="@0.0.2:")
+    depends_on("gotcha@1.0.5", when="@0.0.2:")
+    depends_on("gotcha@1.0.6", when="@0.0.4")
+    depends_on("gotcha@1.0.7", when="@0.0.5")
+    depends_on("gotcha@1.0.7", when="@develop")
     depends_on("catch2@3.0.1:")
 
     depends_on("mpi", when="+mpi")
 
     def cmake_args(self):
-        return [self.define_from_variant("BUILD_WITH_MPI", "mpi")]
+        if self.spec.satisfies("@0.0.4:"):
+            return [self.define_from_variant("BRAHMA_BUILD_WITH_MPI", "mpi")]
+        else:
+            return [self.define_from_variant("BUILD_WITH_MPI", "mpi")]
```

Comment:
FWIW. This could be tightened up, e.g.,

```suggestion
        prefix = "BRAHMA_" if self.spec.satisfies("@0.0.4:") else ""
        return [self.define_from_variant(f"{prefix}BUILD_WITH_MPI", "mpi")]
```